ontopy/ontology.py

```diff
@@ -1083,9 +1095,13 @@ def save(
                 super().save(tmpfile, format="ntriples", **kwargs)
                 graph = rdflib.Graph()
                 graph.parse(tmpfile, format="ntriples")
-                graph.namespace_manager.bind(
-                    "", rdflib.Namespace(self.base_iri)
-                )
+
+                # Add common namespaces unknown to rdflib
```

Comment:
Looks like we add all namespaces in the dict and override what is potentiallyin rdflib, no?

Reply:
Yes, that was the intention to allow the user to pontially override rdflib defaults.

Reply:
OK, then update the comment to reflect this.